Task1/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace Task1
+
+{
+  class Program
+  {
+    static void Main(string[] args)
+    {
+      // Explicit conversions
+
+      var numb6 = 8195.5; // automatically assigns a double type 
+      long a;
+      int b;
+      short c;
+      byte d;
+
+      a = (long)numb6;
+      b = (int)numb6;
+      c = (short)numb6;
+      d = (byte)numb6;
+
+      // Implicit conversions
+
+      byte numb1 = 255;
+      short numb2 = numb1; // byte -> short
```

Comment:
` byte -> short` what does this comment mean?
You do not need to confuse others developers. We have to add some comments where it is really required.